packages/react-native/Libraries/AppDelegate/RCTAppDelegate.h

```diff
@@ -61,6 +62,10 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong, nullable) RCTBridge *bridge;
 @property (nonatomic, strong, nullable) NSString *moduleName;
 @property (nonatomic, strong, nullable) NSDictionary *initialProps;
+@property (nonatomic, strong, nonnull) RCTRootViewFactory* rootViewFactory;
+
+@property (nonatomic, strong, nullable) RCTSurfacePresenterBridgeAdapter *bridgeAdapter;
```

Comment:
I don't think this should be strong. The object that retains the reference is the `RCTRootViewFactory`, no?

Reply:
Good catch, I've removed `strong` from  bridge and bridgeAdapter 👍🏻